src/compiler/scala/tools/nsc/typechecker/Adaptations.scala

```diff
@@ -67,21 +67,14 @@ trait Adaptations {
       }
 
       if (args.isEmpty) {
-        if (settings.future)
-          context.error(t.pos, adaptWarningMessage("Adaptation of argument list by inserting () has been removed.", showAdaptation = false))
-        else {
-          val msg = "Adaptation of argument list by inserting () is deprecated: " + (
-          if (isLeakyTarget) "leaky (Object-receiving) target makes this especially dangerous."
-          else "this is unlikely to be what you want.")
-          context.deprecationWarning(t.pos, t.symbol, adaptWarningMessage(msg), "2.11.0")
-        }
+        context.error(t.pos, adaptWarningMessage("adaptation of argument list by inserting () has been removed.", showAdaptation = false))
       } else if (settings.warnAdaptedArgs)
         context.warning(t.pos, adaptWarningMessage(
-          s"Adapting argument list by creating a ${args.size}-tuple: this may not be what you want.")
+          s"adapting argument list by creating a ${args.size}-tuple: this may not be what you want.")
```

Comment:
Perhaps for uniformity of tone with the error,

`adaptation of argument list by creation of an N-tuple`

or

`adaptation of argument list having created an N-tuple`

Reply:
How about this?

```
t8610.scala:7: warning: adapted the argument list to the expected 2-tuple: fix the code by adding the additional parens instead
         signature: X.f(p: (Int, Int)): Int
   given arguments: 3, 4
  after adaptation: X.f((3, 4): (Int, Int))
```

Reply:
:+1: